src/frontend/src/views/GroupMembersPage/components/MembersTab/components/InviteModal/index.tsx

```diff
@@ -0,0 +1,132 @@
+  const [invalidAddresses, setInvalidAddresses] = useState<string[]>([]);
+  // @ts-expect-error remove when api call finished
+  const [shouldValidateOnChange, setShouldValidateOnChange] = // eslint-disable-line @typescript-eslint/no-unused-vars
+    useState<boolean>(false);
+  const [isNotificationOpen, setIsNotificationOpen] = useState<boolean>(false);
+
+  const validate = (): boolean => {
+    const addresses = inputValue.trim().split(INPUT_DELIMITERS);
+    const newInvalidAddresses = compact(
+      addresses.filter((address) => !address.match(EMAIL_REGEX))
+    );
+
+    setInvalidAddresses(newInvalidAddresses);
+
+    const hasMoreThan50 = addresses.length > 50;
+    setHasMoreThan50Invites(hasMoreThan50);
+
+    // after they click the button to send invites once, we should debounce validate on
+    // all subsequent changes
+    setShouldValidateOnChange(true);
+
+    return (
+      !hasMoreThan50 && newInvalidAddresses.length === 0 && addresses.length > 0
+    );
+  };
+
+  const onInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
+    setInputValue(value);
+  };
+
+  // const title = (
+  //   <StyledSpan>
+  //     Invite to <B>Santa Clara County</B>
+  //   </StyledSpan>
+  // );
+
+  const instructions = [
+    <span key={0}>You can send a maximum of 50 invitations at a time.</span>,
+    <span key={1}>Separate emails by tabs, commas, or enter one per row.</span>,
+    <span key={2}>
+      Members have full access to group samples and phyogenetic trees; they are
+      able to build trees, upload, download, edit, and delete data.
+    </span>,
+  ];
+
+  const inputIntent = invalidAddresses.length > 0 ? "error" : "default";
+
+  const handleFormSubmit = () => {
+    const areAllAddressesValid = validate();
+    if (!areAllAddressesValid) return;
+
+    // TODO (mlila): api call
+  };
+
+  return (
+    <>
+      <SentNotification
+        numSent={5} // TODO (mlila): update after API call return
+        onDismiss={() => setIsNotificationOpen(false)}
+        open={isNotificationOpen}
+      />
+      <Dialog open onClose={onClose} sdsSize="s">
+        <DialogTitle title="Invite to Santa Clara County" onClose={onClose} />{" "}
+        {/* TODO (mlila): make group name bold after sds dialog allows ReactNode */}
+        <StyledDialogContent>
+          <StyledInstructions
+            title="Instructions"
+            titleSize="s"
+            bodySize="xs"
+            items={instructions}
+          />
+          <StyledInputText
+            id="invite-email-input"
+            sdsType="textArea"
+            label="Emails"
+            onChange={onInputChange}
+            placeholder="e.g. userone@domain.com, usertwo@domain.com"
+            intent={inputIntent}
+            value={inputValue}
+            maxRows={4}
+          />
+          {hasMoreThan50Invites && (
+            <StyledCallout intent="error">
+              <B>You can’t send more than 50 invites at a time. </B>
```

Comment:
Do we use directional apostrophes/quotes? For some reason I thought we just used straight ones. I don't have a strong opinion here, just want to verify this was intentional

Reply:
I literally copy paste the text from Figma, so whatever Janeece uses is what goes into the app